packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -648,6 +681,7 @@ public void setBorderStyle(@Nullable String style) {
 
   public void setSpanned(Spannable spanned) {
     mSpanned = spanned;
+    mShouldAdjustSpannableFontSize = true;
```

Comment:
QQ: Should we set this to true also inside:

- `setNumberOfLines`
- `setMinimumFontSize`

and theoretically also inside the following overrides (which we'll have to add):

- `setIncludeFontPadding()`
- `setBreakStrategy()`
- `setHyphenationFrequency()`

As they all contribute to the computation of `adjustSpannableFontToFit`

Reply:
I believe you're right, done in 1c879ac